src/test/java/com/thealgorithms/maths/LongDivisionTest.java

```diff
@@ -4,7 +4,7 @@
 
 import org.junit.jupiter.api.Test;
 
-public class LongDivisionTest {
+class LongDivisionTest {
```

Comment:
What the benefits to remove `public`?

Reply:
This was a code smell issue sonarlint. Starting from Junit 5 classes do not need to be public.